material/problems/VectorGeometry/PolygonNormal/PolygonNormal.tex

```diff
@@ -1,65 +1,125 @@
 \providecommand{\isolatedBuild}[1]{#1}% fallback definition lets this file build normally
 \isolatedBuild
 {
-\documentclass[11pt,letterpaper]{book}
-\usepackage{import}
-
-% This file must be found via the TEXINPUTS environment variable.
-\input{handout_helpers.tex}
-
-\pagestyle{empty}
-\pagenumbering{arabic}
-
-\begin{document}
-\HandoutHeader{Vector Equations and Geometry}
-\normalsize
-\vspace{-1.5pc}
-\\[0.0pc]
-%
-%\begin{enumerate}
-%\item
-\textColorBold{darkerBlue}{Polygon Meshes in Engineering and Graphics}
-\\[0.5pc]
+  \documentclass[11pt,letterpaper]{book}
+  \usepackage{import}
+  %
+  % This file must be found via the TEXINPUTS environment variable.
+  \input{handout_helpers.tex}
+  %
+  \pagestyle{empty}
+  \pagenumbering{arabic}
+  %
+  \begin{document}
+  \HandoutHeader{Vector Equations and Geometry}
+  \normalsize
+  \vspace{-1.5pc}
+  \\[0.0pc]
+  \textColorBold{darkerBlue}{Polygon Meshes in Engineering and Graphics}
+  \\[0.5pc]
 }
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

Comment:
Please replace the header above using the header style found in the other examples (`VectorBasics/BasisIndependentDotCross/`,  `VectorBasics/ResultantForce1/`, etc).